Makefile.inc

```diff
@@ -13,3 +13,9 @@ clusters:
 
 deploy: clusters
 	$(SCRIPTS_DIR)/deploy.sh --globalnet $(globalnet) --deploytool $(deploytool) $(DEPLOY_ARGS)
+
+ifeq (go.mod,$(wildcard go.mod))
```

Comment:
Why wildcard here? It's only for expansion of patterns AFAICT whilst `go.mod` isn't one.
Also what's the point of this comparison?

Reply:
It’s an existence check: `$(wildcard go.mod)` expands to `go.mod` if the file exists, empty otherwise.

Reply:
ah i see a bit confusing but cool :)
maybe a comment would've been nice here ;)

Reply:
Done!